### DISCUSSION TITLE
Panda T-shirt logo designs!

### DISCUSSION THREAD

**Discussion by JessicaSachs** [faker-js/faker#1221] [General]
## 👕 Faker T-Shirt + Swag Discussions

I've been hunting and working with different vendors to get our beloved Panda on a Faker.js T-Shirt.

<img src="https://user-images.githubusercontent.com/2801156/182285178-cce62e52-922e-4876-8b5f-164615cb7be4.png" width="150"/>

We wanted some help designing T-Shirt friendly logos from a real printing shop to reduce the number of colors, etc. I reached out to a couple of vendors and will upload some of the designs below when they're ready for review.

The only decision we made was that the majority of maintainers decided to go with a **dark** T-Shirt for now instead of a white one (although, ideally, @Shinigami92 can get his light-colored Faker shirt at some point!).

**Comment by ejcheng** [comment]
Can't wait to see the designs!

**Comment by JessicaSachs** [comment]
So here is what the design agency sent me the other day. I don't love them.

<details>
<summary>Black, front + back</summary>

![123_1](https://user-images.githubusercontent.com/2801156/187082708-d561b440-9b0c-4247-a755-a951a0b32538.jpg)
![123_1](https://user-images.githubusercontent.com/2801156/187082699-d23fc3eb-8f74-46de-aa04-497510b0542e.jpg)

</details>

<details>
<summary>White, front + back</summary>

![123_1](https://user-images.githubusercontent.com/2801156/187082717-a14ea87d-6d9d-471c-8c6d-649f21d5f22d.jpg)
![123_1](https://user-images.githubusercontent.com/2801156/187082711-20e73715-a785-494c-9a0a-a2c684a6af53.jpg)
</details>

**Reply by Shinigami92** [reply]
Yeah I don't really like them either
The title "Faker" is to near to the logo and the back text should be one of the slogans: #1263 

Also the black shirt doesn't fit that nicely to the already mostly blackish logo :thinking: 
We need a second iteration

**Reply by ejcheng** [reply]
> Yeah I don't really like them either The title "Faker" is to near to the logo and the back text should be one of the slogans: #1263
> 
> Also the black shirt doesn't fit that nicely to the already mostly blackish logo 🤔 We need a second iteration

Yeah, I think it would look better if the entire T-shirt was white and the `npm install` text was black and maybe moved down a little bit.

**Reply by ST-DDT** [reply]
What is bad about these designs?
If the "missing" black border is the issue, then maybe add an additional white one or the greenish one from discord.
![grafik](https://user-images.githubusercontent.com/1579362/187287991-b099e367-78b6-447b-9153-1b8c8b8c6f7a.png)
![grafik](https://user-images.githubusercontent.com/1579362/187288145-2e0e8bd3-ac95-4130-86dd-2307a202f20a.png)

**Reply by Shinigami92** [reply]
Yeah I like the one with white additional border

**Reply by ejcheng** [reply]
> What is bad about these designs? If the "missing" black border is the issue, then maybe add an additional white one or the greenish one from discord. ![grafik](https://user-images.githubusercontent.com/1579362/187287991-b099e367-78b6-447b-9153-1b8c8b8c6f7a.png) ![grafik](https://user-images.githubusercontent.com/1579362/187288145-2e0e8bd3-ac95-4130-86dd-2307a202f20a.png)

Oooh, white border looks great

**Comment by agritheory** [comment]
I'm new to this library and community - most of my needs have been met previously by python packages. I feel like there's an opportunity to make a very self-aware joke here about being a "faker". I think the vast majority of developers deal with some kind of imposter syndrome so having the opportunity to wear a tshirt that says "faker" on it represents an inside joke of a sort I personally wouldn't be able to resist.

Assuming I'm not alone in this line of thinking, I would want a tshirt where "faker" or "FAKER" is featured, no disrespect to Panda. With the qualification that I have design ideas but not design skills...

![faker](https://user-images.githubusercontent.com/18033666/188183571-49c3ba30-a44b-4cfd-8078-c1f414079986.PNG)

**Reply by Shinigami92** [reply]
I like the uppercase FAKER, the border-rectangle and the logo below
I only miss the opportunity of a slogan
Also what would be the front side?

At least this could be one option of a shirt :slightly_smiling_face:

**Reply by ST-DDT** [reply]
As for the slogan:  

> There is no turning back

🤣

**Reply by ST-DDT** [reply]
I would put the logo on the front and the slogan on the back though.

Then you could also combine the slogan with the install command.

**Reply by agritheory** [reply]
> At least this could be one option of a shirt

To be clear I don't think this is the kind of thing that everybody would want to wear, but it would my choice, mostly for the joke. To increase the self-awareness, the logo-with-border could be rendered with a rubber stamp effect. If the design leans to far into the I-have-imposter-syndrome ethos, it gets more uncomfortable and exclusionary and pedantic, which should _not_ be a goal, even if its intended for humor.

**Reply by JessicaSachs** [reply]
We're gonna get on the phone later this week and pair in real-time. I put together maybe 13 different designs so we'll see how we like them. Join the [Discord](https://chat.fakerjs.dev).
